File: app/src/main/java/me/aluceps/sandbox/api/service/ConnpassService.java
package me.aluceps.sandbox.api.service;

import android.support.annotation.NonNull;

import io.reactivex.Single;
import me.aluceps.sandbox.model.ConnpassEvent;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface ConnpassService {

    @GET("event/")
    Single<ConnpassEvent> events(
            @Query("start") @NonNull int start,
            @Query("count") @NonNull int count
    );
}

File: app/src/main/java/me/aluceps/sandbox/api/service/ConnpassService.kt
package me.aluceps.sandbox.api.service

import io.reactivex.Single
import me.aluceps.sandbox.model.ConnpassEvent
import retrofit2.http.GET
import retrofit2.http.Query

interface ConnpassService {

    @GET("event/")
    fun events(
            @Query("start") start: Int,
            @Query("count") count: Int
    ): Single<ConnpassEvent>
}
